src/utils.js

```diff
@@ -0,0 +1,122 @@
+import Todo from './todo.js';
+
+export default class Methods {
+static setLocalStorageData = (todo) => {
+  const item = JSON.stringify(todo);
+  localStorage.setItem('todoList', item);
+};
+
+static getLocalStorageData = () => {
+  let todoList;
+
+  if (JSON.parse(localStorage.getItem('todoList')) === null) {
+    todoList = [];
+  } else {
+    todoList = JSON.parse(localStorage.getItem('todoList'));
+  }
+
+  return todoList;
+};
+
+static reassignIndex = (todoList) => {
+  todoList.forEach((item, i) => {
+    item.index = i + 1;
+  });
+}
+
+static deleteTask = (id) => {
+  let todoList = this.getLocalStorageData();
+  const itemToDelete = todoList[id];
+
+  todoList = todoList.filter((item) => item !== itemToDelete);
+
+  this.reassignIndex(todoList);
+  this.setLocalStorageData(todoList);
+};
+
+static updateTaskInput = (newDesc, id) => {
+  const todoList = this.getLocalStorageData();
+  const itemToUpdate = todoList[id];
+
+  todoList.forEach((item) => {
+    if (item === itemToUpdate) {
+      item.description = newDesc;
+    }
+  });
+
+  this.setLocalStorageData(todoList);
+  this.showTodoItems();
+};
+
+static addBtnRemoveEvent = () => {
+  document.querySelectorAll('.trash-can').forEach((button) => button.addEventListener('click', (event) => {
+    event.preventDefault();
+    let id;
+    if (button.id > 0) {
+      id = button.id - 1;
+    } else {
+      id = 0;
+    }
+    this.deleteTask(id);
+    this.showTodoItems();
+  }));
+};
```

Comment:
- [ ] [OPTIONAL] Good job so far 👍 If It would be nice if you could also implement the delete function like [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.) 😃 

![image](https://user-images.githubusercontent.com/63915024/199526066-b791c21f-649f-49bb-b9c4-e9b5d6be3f0e.png)